JASP-Engine/JASP/po/R-nl.po

```diff
@@ -933,14 +1315,94 @@ msgstr ""
 "de auditwaardes, een indicator van de sterkte van de lineaire relatie tussen "
 "de twee variabelen."
 
-msgid ""
-"Your sample size is larger than your population size. Cannot take a sample "
-"larger than the population."
-msgstr ""
-"Uw steekproefgrootte is groter dan de populatiegrootte. Kan geen steekproef "
-"nemen groter dan de populatie."
+msgid "<b>percentage</b>"
+msgstr "<b>percentage</b>"
 
-msgid ""
+msgid "<b>amount</b>"
+msgstr "<b>hoeveelheid</b>"
+
+msgid "<u>Planning</u>"
+msgstr "<u>Planning</u>"
+
+msgid "<u>Selection</u>"
+msgstr "<u>Selectie</u>"
+
+msgid "<u>Evaluation</u>"
+msgstr "<u>Evaluatie</u>"
+
+msgid ""
+"Analysis not possible: Your materiality is higher than, or equal to the "
+"total value of the observations."
+msgstr ""
+"Analyse niet mogelijk: De materialiteit is groter of gelijk aan dan de "
+"totale waarde van de observaties."
+
+msgid ""
+"Analysis not possible: Your expected errors are higher than materiality."
+msgstr ""
+"Analyse niet mogelijk: De verwachte fouten zijn groter dan de materialiteit."
+
+msgid ""
+"The detection risk is equal to or higher than 100%. Please re-specify your "
+"custom values for the Inherent risk and/or Control risk, or the confidence."
+msgstr ""
+"Het detectie risico is hoger dan 100%. Herspecificeer de aangepaste waarden "
+"voor het Inherente risico en/of het Intern Beheersingsrisico."
+
+msgid "Selection summary"
+msgstr "Selectie samenvatting"
+
+msgid ""
+"Your sample size is larger than (or equal to) your population size. Cannot "
+"take a sample larger than the population."
+msgstr ""
+"Uw steekproefgrootte is groter of gelijk aan de populatiegrootte. Kan geen "
+"steekproef nemen groter dan de populatie."
+
+msgid "Your sample size must be larger than 1."
+msgstr "Uw steekproefgrootte moet groter dan 1 zijn."
+
+msgid ""
+"Your must specify unique record ID's. The row numbers of the data set are "
+"sufficient."
+msgstr ""
+"U moet unieke transactie ID's specificeren. De rijnummers van de data set "
```

Comment:
Hebben we eigenlijk niet iets moois voor "data set"?

Reply:
gegevens?